vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -99,16 +112,72 @@ public RedirectView createOrUpdateExhibition(HttpServletRequest request,
         exhibition.setStartSpace(startSpace);
         exhibition.setTitle(title);
         exhibition.setMode(exhibitMode);
-        exhibitManager.updateExhibitionLanguages(exhibition,languages,defaultLanguage);
-    
-        if(exhibitMode.equals(ExhibitionModes.OFFLINE) && !customMessage.equals(ExhibitionModes.OFFLINE.getValue())) {
+
+        exhibitManager.updateExhibitionLanguages(exhibition, languages, defaultLanguage);
+
+        if (exhibitMode.equals(ExhibitionModes.OFFLINE) && !customMessage.equals(ExhibitionModes.OFFLINE.getValue())) {
 
             exhibition.setCustomMessage(customMessage);
         }
+
+        exhibition = (Exhibition) exhibitManager.storeExhibition(exhibition);
+        attributes.addAllAttributes(addSuccessAttributes(exhibition));
+
+        return new RedirectView(request.getContextPath() + "/staff/exhibit/config");
+    }
+    
+    /**
+     * To create or update the default link images.
+     * 
+     * @param image - the MultipartFile representing the link image to be uploaded
+     * @param linkType - the type of the link for which the default image must be set
+     * @param attributes - the RedirectAttributes object to add flash attributes
+     * @return - a RedirectView to the exhibition configuration page
+     * @throws IOException if an input or output error occurs
+     */
+    @RequestMapping(value = "/staff/exhibit/config/link/defaultImage", method = RequestMethod.POST)
+    public RedirectView createOrUpdateLinkImage(HttpServletRequest request,
+            @RequestParam(name = "image", required = false) MultipartFile image,
+            @RequestParam(name="linkType") String linkType,
+            RedirectAttributes attributes) throws IOException {
+        Exhibition exhibition = (Exhibition) exhibitManager.getStartExhibition();       
+        if(linkType.equals("space")) {
+            IVSImage spaceDefaultImage = imageService.storeImage(image.getBytes(), image.getOriginalFilename()); 
+            exhibition.setSpaceLinkDefaultImage(spaceDefaultImage);
+        }
+        else if(linkType.equals("module")) {
+            IVSImage moduleDefaultImage = imageService.storeImage(image.getBytes(), image.getOriginalFilename());            
+            exhibition.setModuleLinkDefaultImage(moduleDefaultImage);
+        }
+        else if(linkType.equals("external")) {
+            IVSImage externalLinkDefaultImage = imageService.storeImage(image.getBytes(), image.getOriginalFilename());         
+            exhibition.setExternalLinkDefaultImage(externalLinkDefaultImage);
+        }
+        else {
+            attributes.addAttribute("alertType", "danger");
+            attributes.addAttribute("showAlert", "true");
+            attributes.addAttribute("message", "Couldn't save the default image");
+            return new RedirectView(request.getContextPath() + "/staff/exhibit/config");
```

Comment:
context path should not be needed here. if the method returns `String`, you can just return `redirect:/path...`.